### ISSUE TITLE
unwatch() and close() after a delay from watching does not allow process to exit in OSX 

### ISSUE THREAD

**Issue by Zugwalt** [paulmillr/chokidar#218]
After a few milliseconds, unwatching and closing doesn't allow process to exit in OSX.

```javascript
var chokidar = require('chokidar');

console.log('watching',__dirname);
var watcher = chokidar.watch(__dirname);

setTimeout(function() {
    console.log('unwatching');
    watcher.unwatch(__dirname);
    console.log('closing');
    watcher.close();
}, 1000);
````

**Comment by es128** [comment]
Dupe of https://github.com/pipobscure/fsevents/issues/50

**Comment by es128** [comment]
Fixed in fsevents v0.3.5
